podium-uaa/src/main/java/nl/thehyve/podium/web/rest/UserResource.java

```diff
@@ -134,7 +134,7 @@
         log.debug("REST request to update User : {}", userData);
         userService.updateUser(userData);
         return userService.getUserByUuid(userData.getUuid())
-            .map(ManagedUserVM::new)
+            .map(User::toManagedUserVM)
```

Comment:
The User to ManagedUser should be added to the UserMapper as well. Dont worry for now I guess